app/src/main/java/com/github/weg_li_android/MainActivity.kt

```diff
@@ -25,12 +59,177 @@ class MainActivity : AppCompatActivity() {
             ViewModelFactory(ApiHelper(ApiServiceImpl()))
         ).get(MainViewModel::class.java)
 
+        setupPhotoCard(this)
+
         setupCarTypeSpinner()
         setupViolationSpinner()
         durationText.setOnClickListener {
         }
 
         sendButton.setOnClickListener { mainViewModel.sendReport() }
+
+        val licenseObserver = Observer<Report> { newName ->
+            if(newName.license != "") {
+                status_image.setImageResource(R.drawable.ic_baseline_check_circle_24)
```

Comment:
In Kotlin it's better to name it "statusImage" instead of "status_image", since we're using these names just like here in code.